contrib/great_expectations_semantic_types_expectations/great_expectations_semantic_types_expectations/expectations/expect_column_values_to_be_valid_city_name.py

```diff
@@ -15,10 +15,7 @@ def is_valid_city_name(city: str):
     list_of_city_names = [item["name"] for item in list_of_cities]
     if len(city) > 54 or type(city) != str:  # noqa: E721
         return False
-    elif city in list_of_city_names:
-        return True
-    else:
-        return False
+    return city in list_of_city_names
```

Comment:
This one is interesting since it's breaking off the end of an `if` statement. I kind of like the original version better here but 🤷 .

Reply:
I agree - I added an ignore.